x-pack/plugins/security_solution/public/timelines/components/timeline/expandable_event/index.tsx

```diff
@@ -68,12 +101,20 @@ export const ExpandableEvent = React.memo<Props>(
 
     return (
       <>
-        <EuiText>{message}</EuiText>
+        {message && (
```

Comment:
Thank you ❤️

Reply:
can we just improve the height of the one line content as the current solution creates too much of the space between message and the tabs?
![image](https://user-images.githubusercontent.com/5188868/101545689-eccb4c00-39a7-11eb-9c71-96cb6de1887d.png)